lib/salus/repo.rb

```diff
@@ -95,5 +95,18 @@ class Repo
     def initialize(path_to_repo = nil)
       @path_to_repo = path_to_repo
     end
+
+    ##
+    # @param [Array] args CLI rg command and options to run
+    # @return [Array<String>] Relative path of files returned
```

Comment:
This comment says ` # @return [Array<String>] Relative path of files returned`. But, I think the method is returning absolute paths. Am I right? 🤔 

Can we please change the comment so that it is consistent with the method body?